README.md

```diff
@@ -33,118 +36,123 @@ Use the [documentation](https://docs.altinn.studio/teknologi/altinnstudio/) to g
 
 1. Navigate to the `development` folder in the altinn-studio repo
 
-```cmd
-cd src/development
-```
+   ```bash
+   cd src/development
+   ```
 
 2. Start the loadbalancer container that routes between the local platform services and the app
 
-```cmd
-docker-compose up -d --build
-```
+   ```bash
+   docker-compose up -d --build
+   ```
 
 3. Set path to app folder in local platform services:
- - Open `appSettings.json` in the `LocalTest` folder, f.ex. in Visual Studio Code
+   - Open `appSettings.json` in the `LocalTest` folder, f.ex. in Visual Studio Code
 
-```cmd
-cd LocalTest
-code appSettings.json
-```
+   ```bash
+   cd LocalTest
+   code appSettings.json
+   ```
 
- - Change the setting `"AppRepsitoryBasePath"` to the full path to your app on the disk. Save the file.
+   - Change the setting `"AppRepsitoryBasePath"` to the full path to your app on the disk. Save the file.
 
 4. Start the local platform services (make sure you are in the LocalTest folder)
 
-```cmd
-dotnet run
-```
+   ```bash
+   dotnet run
+   ```
 
 5. Navigate to the app folder (specified in the step above)
 
-```cmd
-cd \.\<path to app on disk>
-```
+   ```bash
+   cd /<path to app on disk>
+   ```
 
-- If you need to debug (or run locally) the app front-end:
-  - Open the file `views/Home/Index.cshtml` and change the line
+   - If you need to debug (or run locally) the app front-end:
+     - Open the file `views/Home/Index.cshtml` and change the line
 
-  ```html
-   <script src="https://altinncdn.no/toolkits/altinn-app-frontend/2/altinn-app-frontend.js"></script>
-    ```
+     ```html
+     <script src="https://altinncdn.no/toolkits/altinn-app-frontend/2/altinn-app-frontend.js"></script>
+     ```
 
-    to
+     to
 
-    ```html
-     <script src="http://localhost:8080/altinn-app-frontend.js"></script>
-    ```
+     ```html
+      <script src="http://localhost:8080/altinn-app-frontend.js"></script>
+     ```
 
-    - Build and run the runtime front-end project locally (`altinn-studio/src/Altinn.Apps/AppFrontend/react`):
-    ```cmd
-    npm install # only needed first time, or when dependencies are updated
-    npm run install-deps # only needed first time, or when dependencies are updated
-    cd altinn-app-frontend
-    npm start
-    ```
+     - Build and run the runtime front-end project locally (`altinn-studio/src/Altinn.Apps/AppFrontend/react`):
+  
+     ```bash
+     npm install # only needed first time, or when dependencies are updated
+     npm run install-deps # only needed first time, or when dependencies are updated
+     cd altinn-app-frontend
+     npm start
+     ```
 
 6. Start the app locally
 
-```cmd
-dotnet run -p App.csproj
-```
+  ```bash
+  dotnet run -p App.csproj
+  ```
 
-The app and local platform services are now running locally. The app can be accessed on altinn3local.no.
+The app and local platform services are now running locally. If you have configured your hosts. The app can be accessed on altinn3local.no.
```

Comment:
Suggestion:
If you have [configured your hosts](link to relevant section describing this), the app can be accessed...